docs/source/using/host-folders.adoc

```diff
@@ -0,0 +1,225 @@
+The share is currently not mounted.
+
+NOTE: The remote path must be reachable from within the VM.
+In the example above _192.168.1.82_ is the IP of host within the LAN which is one options one can use.
+You can use `ifconfig` (or `Get-NetIPAddress | Format-Table` on Windows) to determine a routable IP address.
+
+[[adding-host-folders]]
+=== Adding host folders
+
+The link:../command-ref/minishift_hostfolder_add{outfilesuffix}[`hostfolder add`] command allows you define a new host folder.
+This in an interactive process which queries the relevant details for a host folder based on CIFS.
+<<adding-cifs-hostfolder>> shows an example.
+
+[[adding-cifs-hostfolder]]
+.Adding a CIFS based hostfolder
+----
+$ minishift hostfolder add mac // <1>
+UNC path: //192.168.99.1/MAC   // <2>
+Mountpoint [/mnt/sda1/mac]:    // <3>
+Username: john                 // <4>
+Password: [HIDDEN]             // <5>
+Domain:                        // <6>
+Added: mac
+----
+<1> (Required) Actual `hostfolder add` command, specifying a hostfolder with a name of _mac_.
+<2> (Required) The UNC path for the share.
+<3> The mount point within the VM. The default is _/mnt/sda1/<host folder name>_.
+<4> (Required) The username for the CIFS share.
+<5> (Required) The password for the CIFS share.
+<6> The domain of the share. Often this can be left blank, but for example on Windows, when your account is linked to a Microsoft account, you must use the Microsoft account email address as username as well as your machine name as displayed by _$env:COMPUTERNAME_ as domain.
+
+TIP: On Windows hosts the `hostfolder add` command also provides a _users-share_ option.
+When specified no UNC path needs to be specified and the _C:\Users_ is assumed.
+
+WARNING: When you use the Boot2Docker ISO along with the VirtualBox driver, VirtualBox guest additions are automatically enabled and occupy the _/Users_ mountpoint.
+
+==== Instance specific host folders
+
+Per default host folder definitions are persistent similar to options specified using the `<<managing-minishift#persistent-configuration, minishift config>>` command.
+This means that these host folder definitions will survive the deletion and subsequent re-creation of a Minishift VM.
+
+In some cases you might want to define a host folder just for a specific Minishift instance. To do so you can use the _instance-only_ flag of the link:../command-ref/minishift_hostfolder_add{outfilesuffix}[`hostfolder add`] command.
+Host folder definition created with the _instance-only_ flag will be removed together with any other instance specific state during a `<<managing-minishift#minishift-delete,minishift delete>>`.
+
+[[mounting-host-folders]]
+=== Mounting host folders
+
+Once you have added your host folder definitions (see <<adding-cifs-hostfolder>>), the link:../command-ref/minishift_hostfolder_mount{outfilesuffix}[`hostfolder mount`] command allows you to mount a host folder by its name:
```

Comment:
+1 
Suggestion: Can we link the tag <<adding-cifs-hostfolder>> to the text 'added your host folder', that way we do not need to add the relevant information in parenthesis